app/src/main/java/com/hamdan/forzenbook/view/login/MainLogin.kt

```diff
@@ -45,70 +51,95 @@ fun MainLoginContent(
             contentDescription = "Yellow lion"
```

Comment:
Content descriptions are user facing. Strings.xml

Reply:
ok didn't realize, it's updated now